x-pack/plugin/sql/build.gradle

```diff
@@ -10,6 +10,12 @@ esplugin {
     extendedPlugins = ['x-pack-core', 'lang-painless']
 }
 
+ext {
+    // SQL dependency versions
+    jlineVersion="3.10.0"
```

Comment:
We have a way to express common versions in `buildSrc/version.proeprties` you can then access these with `${versions.jlineVersion}` . I think that's better than relying on Gradle to magically resolve these in sub-projects.

Reply:
My concern is that this is a project used by xpack and I'd like to keep things separately in `buildSrc` so there's no confusion. As far as I can tell, all the versions declared there are for OSS dependencies only...